src/import/stream.rs

```diff
@@ -143,25 +143,25 @@ impl RangeIterator {
         };
 
         // Seek to the first valid range.
-        res.seek_next();
```

Comment:
Is this intended to be an unwrap() here or just to avoid changing too many code?